### DISCUSSION TITLE
Teams Notification after Actions

### DISCUSSION THREAD

**Discussion by MRMMorten** [microsoft/AL-Go#213] [Ideas]
Hello, 

i think it would be a great idea to use "notify-microsoft-teams" as action 
to get the Result informations directly into Teams.

What do you think about it?

**Comment by curateideas** [comment]
Seems GitHub has the support for actions in their Teams App already in the backlog:

![image](https://user-images.githubusercontent.com/58227881/191784340-425b0b58-6ac0-4955-851b-1ba447ded254.png)

https://github.com/integrations/microsoft-teams#upcoming-features-and-roadmap
